src/_includes/platforms/python/request-data.md

```diff
@@ -1 +1,3 @@
 Request data is attached to all events: **HTTP method, URL, headers, form data, JSON payloads**. Sentry excludes raw bodies and multipart file uploads. Sentry also excludes personally identifiable information (such as user ids, usernames, cookies, authorization headers, IP addresses) unless you set ``send_default_pii`` to ``True``.
+
+Each request has a separated scope. Setting a tag during import time will set a global tag, while setting a tag within a view will attach the tag only to events sent as part of that request lifecycle.
```

Comment:
```suggestion
Each request has a separate scope. Setting a tag during import time will set a global tag, while setting a tag within a view will attach the tag only to events sent as part of handling an individual request.
```

> Setting a tag during import time

This seems more confusing than clarifying to me. What does it mean to set a tag during import time?

Maybe a code snippet would be clearer?

In both cases it is going to be `scope.set_tag`, the difference being where the `scope` is coming from. On the script's top-level, that's going to be likely the scope attached to the global hub, and in the view code that is going to be the scope created specifically for the execution of the view.